rollbar-cli/package.yaml

```diff
@@ -1,11 +1,11 @@
 name: rollbar-cli
-version: 0.1.0
+version: 0.2.0
```

Comment:
@cptrodolfox taking into account that we are using semantic versioning in the context of this project and we are introducing a breaking change, I think we need to bump the major version

Reply:
@sestrella So the new version should be 1.0.0?